Cop for checking empty raise (#10)
amagimedia/rubocop-amagi: lib/cops/readme.md

CustomCop/DuplicateConstantCop:
  Enabled: false
~~~

### RaiseMessageCop
RaiseMessageCop ensures that there is no empty occurence of raise, i.e it makes sure that "raise" is always accompanied by an argument/message.

### Reason to add this cop

> if we have an occurence of raise without a message or an argument it breaks the flow of the code and the rest of the code becomes unreachable, hence raising "unreachable code" offense on rubocop.

Reviewer: ```suggestion
if we have an occurrence of raise without a message or an argument it breaks the flow of the code and will not know the reason for the exception.
```
